src/elm/Api.elm

```diff
@@ -36,13 +30,6 @@ backendUrl { endpoints } paths queryParams =
         queryParams
 
 
-backendUrl_ : Endpoints -> List String -> List QueryParameter -> String
-backendUrl_ endpoints paths queryParams =
```

Comment:
I'm not sure about this but to upload images and create invites don't we need this?

Reply:
There's a `backendUrl` function right above this one that is exactly the same, except it takes an entire `Shared` instead of just the `Endpoints`

Reply:
oh okay